### PR TITLE
Added windows workflow

### PR DESCRIPTION
**Pull request by hansenms**

### CODE REVIEW

**Comment by dchansen** [comment]
Looks good to me. Is anything missing?

**Comment by hansenms** [comment]
@dchansen almost there, I think. Just making sure that the pipeline actually fails when it should. So, a few more iterations and I will publish the PR in not too long and let you know. Good thing is that we do have a Windows pipeline.

**Comment by hansenms** [comment]
OK @dchansen this should be ready. And @KrisThielemans this includes #134 so we can close that when this is merged.

File: cmake/FindFFTW3.cmake [hunk 1]
```diff
@@ -75,17 +75,16 @@ set(_check_list)
...
-      find_library(${_LIB}_LIBRARY NAMES lib${_lib}-3 lib${_lib}
+      find_library(${_LIB}_LIBRARY lib${_lib}-3 ${_lib}
         HINTS $ENV{FFTW3_ROOT_DIR} PATH_SUFFIXES lib)
       mark_as_advanced(${_LIB}_LIBRARY)
       list(APPEND FFTW3_LIBRARIES ${${_LIB}_LIBRARY})
       list(APPEND _check_list ${_LIB}_LIBRARY)
     endforeach()
 
-    message(STATUS "FFTW3 WINDOWS libraries: " ${FFTW3_LIBRARIES})
+    message("FFTW3 WINDOWS libraries: " ${FFTW3_LIBRARIES})
```
Comment (KrisThielemans) [inline cmake/FindFFTW3.cmake:87]: just checking, why do you remove the `STATUS`? From the CMake doc:
```
(none) or NOTICE
Important message printed to stderr to attract user’s attention.

STATUS
The main interesting messages that project users might be interested in. Ideally these should be concise, no more than a single line, but still informative.
```

Comment (hansenms) [reply cmake/FindFFTW3.cmake]: Yeah, no reason really. I was coping a section from the Gadgetron repo that does this a bit differently. I put it back in.

**Comment by hansenms** [comment]
@dchansen are OK to get this merged so we have some coverage on the Windows side before proceeding?

**Review by dchansen — APPROVED** [review]

**Pull request merged**